sources/packages/web/src/views/institution/student/applicationDetails/ApplicationExceptions.vue

```diff
@@ -31,6 +32,17 @@ export default defineComponent({
     },
   },
   setup(props) {
+    const currentApplicationId = ref<number>();
+
+    onMounted(async () => {
+      // Get current application for the parent application.
+      const currentApplication = await ApplicationService.shared.getApplication(
+        props.applicationId,
+        { loadDynamicData: false, isParentApplication: true },
+      );
```

Comment:
Depending on how we proceed in the upcoming expansion of the history tickets, if this logic is no longer temporary we should be moved to the service to avoid duplication.
We can have a method at the ser ice level as 
`ApplicationService.shared.getCurrentApplicationVersion(parentApplicationId: number)`
Does it make sense?